docs/tutorials/data-modeling-lifecycle/part-3-extending-enterprise-model.md

```diff
@@ -1 +1,274 @@
+
+Olav goes back to Svein Harald, and together they add the following properties to the `Properties` sheet:
+
+| Class                     | Property    | Value Type | Min Count | Max Count |
+|---------------------------|-------------|------------|-----------|-----------|
+| TimeseriesForecastProduct | name        | string     | 1         | 1         |
+| TimeseriesForecastProduct | description | string     | 0         | 1         |
+| TimeseriesForecastProduct | sources     | string     | 0         | Inf       |
+| TimeseriesForecastProduct | low         | timeseries | 1         | 1         |
+| TimeseriesForecastProduct | expected    | timeseries | 1         | 1         |
+| TimeseriesForecastProduct | high        | timeseries | 1         | 1         |
+
+Note that the `sources` property is a list of strings that are used to create the forecast. Olav has checked with
+Lars that this is a good way to capture the input data to the forecast.
+
+## Updating the Spreadsheet (Download Svein Harald's Information spreadsheet)
+
+The finshied spreadsheet with the extension of the Enterprise model is now done.
+
+You can download it [here](../../artifacts/rules/information-addition-svein-harald.xlsx).
+
+## Implementing the Extension
+
+Svein Harald and Olav have now defined all the extensions of the Enterprise model. Olav is happy with the results, and
+leaves it to Svein Harald to get the extension implemented.
+First, Svein Harald uses **NEAT** to convert the spreadsheet he has from information architect to dms architect format.
+He does this by selecting the `Validate Rules` workflow. Note that this will also validate that he has
+written up the spreadsheet correctly. In the `Validate Rules` workflow, Svein Harald selects the `Convert Rules` step
+and sets `Output role format` to `dms_architect`. After running the workflow, Svein Harald can download the converted
+spreadsheet by clicking `exported_rules_DMS_Architect.xlsx`.
+
+<img src="../../artifacts/figs/life_cycle_converter_model_analytic_solution_model.png" height="300">
+
+**NEAT** has given a good out-of-the-box suggestion for how to implement the extension model. However, to ensure that
+the solution model is well aligned with the existing Enterprise model and is performant, Svein Harald
+asks the DMS solution architect, Alice, to help him as he is not an expert in the implementation of the model.
+
+Alice and Svein Harald have a discussion about the new concepts. Alice suggests that they should add an index to the
+`name` and `sources` properties in the `TimeseriesForecastProduct` to ensure that the queries are performant.
+Svein Harald agrees and they add index to the `name` and `sources` properties in the `Properties` sheet.
+
+After the implementation is done, Alice validates the solution model by running the `Validate Rules` workflow with
+the new spreadsheet as input. The validation is successful, and the extension model is ready to be deployed.
```

Comment:
super neat with role definitions in the lifecycle, which leads to clear responsibilities